about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bim online training class</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/a329405765.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="body-overlay hidden"></div>
+    <header>
+        <section class="user-acc">
+            <div class="social-icon">
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-brands fa-facebook"></i>
+            </div>
+            <div class="user">
+                <nav class="nav-user">
+                    <ul>
+                        <li class="logout">Logout</li>
+                        <li class="mypage">My page</li>
+                        <li class="langue">English</li>
+                    </ul>
+                </nav>
+            </div>
+        </section>
+        <section class="main-menu">
+            <a href="./index.html"><img src="./image/logo.png" alt="logo" class="logo" width="45"></a>
+            <nav class="main-nav">
+                <i class="fa-sharp fa-solid fa-bars hamberger"></i>
+                <i class="fa-solid fa-xmark close hidden"></i>
+                <ul class="menu">
+                    <li class="about menu-i"><a href="#">About</a></li>
+                    <li class="class menu-i"><a href="#">Class</a></li>
+                    <li class="join menu-i"><a href="#">Join</a></li>
+                    <li class="teacher menu-i"><a href="#">Teacher</a></li>
+                    <li class="news menu-i"><a href="#">News</a></li>
+                </ul>
+            </nav>
+        </section>
+    </header>
+    <main>
+        <section class="header">
+            <div class="header-overlay"></div>
+            <p class="sub-header">"Hello! Sharing world"</p>
+            <h1>Bim academy training</h1>
+            <p class="description">It feels like the world is moving at a faster pace than ever before. New building regulations, the climate crisis, net zero, ever changing sustainability and safety standards and targets – if any of these are on your mind, BIM Academy can help. Developed and run by BIM Group with over 100 years of built environment knowledge, BIM Academy regularly updates materials to reflect these changes.</p>
+            <div class="contact-info">
+                <p>Please contact us per Email for any further question about BIM Academy!</p>
+                <span class="email">bim-academy@gmail.com</span>
+            </div>
+        </section>
```

Comment:
- [ ] Good :+1: job :100: so far with your design. But I think it would be better if you improve your design to stick as much as possible for to the one from the template( **both MOBILE && DESKTOP**). You should match the same spacing used around & between the content elements, the same font on your page title, text alignment, height, width of different elements, remove the non existing elements, pasddings, margins,... Try to take a look again on the template to improve your design.

| Template Design | Your Design |
| ---------------------- | ---------------- |
| ![Screenshot from 2023-01-05 19-25-10](https://user-images.githubusercontent.com/83042742/210845514-48845d53-6007-44e8-b3eb-e426488eaeb6.png) | ![Screenshot from 2023-01-05 19-37-43](https://user-images.githubusercontent.com/83042742/210845586-9e4d86f3-9a4f-4fc3-b59a-2b49b76e566e.png) |